src/org/zaproxy/zap/extension/ascanrulesBeta/SQLInjectionMySQL.java

```diff
@@ -196,6 +197,10 @@ public void scan(HttpMessage originalMessage, String paramName, String originalP
 			catch (java.net.SocketTimeoutException e) {
 				//to be expected occasionally, if the base query was one that contains some parameters exploiting time based SQL injection?
 				if ( this.debugEnabled ) log.debug("The Base Time Check timed out on ["+msgTimeBaseline.getRequestHeader().getMethod()+"] URL ["+msgTimeBaseline.getRequestHeader().getURI().getURI()+"]");
+			} catch (SocketException ex) {
+				log.info("Caught " + ex.getClass().getName() + " " + ex.getMessage() + 
```

Comment:
Should I also tweak the previous few to debug vs info as well? For consistency.

https://github.com/zaproxy/zap-extensions/pull/228
https://github.com/zaproxy/zap-extensions/pull/221